media-plugins/gst-plugins-qt5/gst-plugins-qt5-1.18.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-good
+
+inherit gstreamer-meson
+
+DESCRIPTION="Qt5 QML video sink plugin for GStreamer"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="+egl wayland +X orc"
+
+REQUIRED_USE="
+	wayland? ( egl )
+"
+
+RDEPEND="
+	>=media-libs/gst-plugins-base-${PV}:${SLOT}[egl?,opengl,wayland?,X?]
+	media-libs/mesa[egl?,X?,${MULTILIB_USEDEP}]
+
+	dev-qt/qtcore
+	dev-qt/qtgui
+	egl? ( dev-qt/qtgui[eglfs] )
+	dev-qt/qtdeclarative
```

Comment:
please keep dependencies sorted, with unconditional dependencies ahead of conditional block.

Reply:
Should be sorted now. Should I have added an empty line before the conditionals? Not sure what is used more often in that case?

Reply:
Deps should be in the order they appear in meson.build, not alphabetically. This is a style difference between a17r and gnome/gstreamer teams. @a17r shouldn't be requesting such changes (in this way that implies it has to get sorted alphabetically) for gstreamer/gnome packages, but I guess technically this was no-ones yet :)

Reply:
It's a style difference between gnome/gstreamer and everyone else.

Reply:
Tbh, I think the order it is in now does make sense. The mesa dependency is just needed to pass through the use flags, so it isn't explicitly in the meson.build and apart from that there is only the qt dependency. So having the 2 use deps on the top and then the meson.build deps after that seems reasonable and is alphabetically sorted as well as pretty much matches the meson.build. Or should I still change anything? I did really appreciate @a17r review, especially as they seem to be involved with qt packages and this package kinda falls in between and it helped resolve quite a few issues!